docs/docs-source/docs/modules/console/pages/index.adoc

```diff
@@ -5,11 +5,8 @@
 
 include::ROOT:partial$include.adoc[]
 
-Available with a Lightbend Platform subscription, Lightbend Console provides deep insights and observability for Cloudflow applications. It helps you verify that your application looks as you expect--with the correct streamlets and the proper connections, each using the desired schema. In the Console, you can watch your application run, check on key performance indicators, such as latency and throughput, and observe the health of each streamlet and the monitors computed on them.
+Available with a Lightbend Platform subscription and on Google Cloud Platform marketplace, Lightbend Console provides deep insights and observability for Cloudflow applications. It helps you verify that your application looks as you expect--with the correct streamlets and the proper connections, each using the desired schema. In the Console, you can watch your application run, check on key performance indicators, such as latency and throughput, and observe the health of each streamlet and the monitors computed on them.
```

Comment:
cc. @RayRoestenburg

Reply:
We are not ready for Google Cloud Platform marketplace yet, so it is better to keep it out for now.

Reply:
Oh, news to me. Thanks! I'll comment it out or remove where necessary. I guess that means the whole section on GCP needs to go too.